src/pages/guides/FAQ/FAQ -11-24

```diff
@@ -0,0 +1,58 @@
+
+
+* How do I **authenticate**?
+
+    The first step in accessing the [Frame.io](http://frame.io/) API is getting authenticated. Create the user authentication credentials and then the access token. Adobe Identity Management Service (IMS) is Adobe’s OAuth 2.0-based identity management feature that supports authentication. There are two steps required for generating the access token:
+
+    * The user has to authorize the application before the token generation. The details for this can be found [here](https://developer.adobe.com/developer-console/docs/guides/authentication/UserAuthentication/IMS/#authorize-request).
+    * The next step is fetching the access token. The above step returns a **code** in the response body. This is used in this step. The exact endpoints to trigger and the parameters to use can be found [here](https://developer.adobe.com/developer-console/docs/guides/authentication/UserAuthentication/IMS/#fetching-access-tokens).
+* Where is the **API reference**?
+    * Go [here](https://developer-stage.adobe.com/frameio/api/current/) to see the Frame V4 API reference pages. 
+* How can I get **help**?
+    * Reach out to the [Frame.io](http://frame.io/) support team for assistance by chatting with us on our [Support Site](https://help.frame.io/en/). Please note that enterprise customers may be directed to their Customer Success Manager.
+* What are the key concepts to the Frame V4 API?
+    * V4 represents an expansion from primarily serving post-production video professionals to a discipline-agnostic platform for creatives and creative managers.  [Frame.io](http://frame.io/) acts as a central canvas for multi-disciplinary teams and solves for key needs for everything from more focus time for creative work to increased visibility into ongoing projects.
+    * **Workflow management**: Centralize, organize, and build all your end-to-end creative workflows at scale leveraging Metadata, Custom Fields, and Collections.
+    * **File transfer and management**: This set of capabilities gives users all the flexibility and power to build the best file management system for their creative process making it easier to quickly get the information they need, and manage all their work-in-progress assets at scale.
+    * **Review and approval**: [Frame.io](http://frame.io/) helps creative teams reduce the total number of revision cycles with advanced review, commenting, and annotation capabilities available across a broad set of media types.
+    * **Sharing and presentation**: Customize and share creative work from one central place.
+    * **Security and governance**: View and manage all users, workspaces, and projects in one place. Take bulk actions and export information to control how you utilize usage data.
+    * **Integrations and APIs**: Native integrations and an open API connect [Frame.io](http://frame.io/) to the systems you already use to unblock sticking points and keep work in flow.
```

Comment:
```suggestion
    * **Open API Standards**: A REST API and an OpenAPI specification connect Frame.io to the systems you already use to unblock custom workflows.
```
We don't support any native integrations yet in our V4 API (at least as part of an officially supported SDK) so I think we may be getting ahead of ourselves here 😉